Storage/src/StorageObject.php

```diff
@@ -1042,6 +1060,10 @@ public function signedUploadUrl($expires, array $options = [])
      *           as part of the signed URL query string. For allowed values,
      *           see [Reference Headers](https://cloud.google.com/storage/docs/xml-api/reference-headers#query).
      *     @type string $version One of "v2" or "v4". **Defaults to** `"v2"`.
+     *     @type bool $virtualHostedStyle If `true`, URL will be of form
+     *           `mybucket.storage.googleapis.com`. If `false`,
+     *           `storage.googleapis.com/mybucket`. Only applies when using v4
+     *           signing. **Defaults to** `false`.
```

Comment:
V2 also supports this if there's a way to get this included.

Reply:
done!